lib/asciidoctor-pdf/theme_loader.rb

```diff
@@ -86,8 +86,10 @@ def load hash, theme_data = nil
   private
 
   def process_entry key, val, data
-    if key == "admonition_icons"
-      data[key] = symbolize_keys val
+    if key.start_with? 'admonition_icon_'
```

Comment:
This changes the theme syntax from:

```yaml
admonition:
  icons:
    tip:
      name: fa-arrows
```

to:

```yaml
admonition:
  icon: # not icons
    tip:
      name: fa-arrows
```

I just want to make sure this is intentional as this differs from the syntax outlined in #121.